### DISCUSSION TITLE
[Feature Request] 1.5 block high sneaking

### DISCUSSION THREAD

**Discussion by unnecessarymb** [ACGaming/UniversalTweaks#314] [Tweak Suggestions]
I could not find a reference mod for this, added in vanilla 1.14. Useful as QOL.

**Comment by magcicada** [comment]
Aqua Acrobatics include this enhancement

**Comment by magcicada** [comment]
![image](https://github.com/ACGaming/UniversalTweaks/assets/45464670/ef820b72-db3c-4a30-93f6-7187692cb45f)
